src/timeMachine/components/view_options/TimeAutoToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {FC, useState} from 'react'
+import {useSelector} from 'react-redux'
+import {SlideToggle, ComponentSize} from '@influxdata/clockface'
+
+import {getXColumnTimeDomainBounds} from 'src/timeMachine/selectors'
+
+interface Props {
+  onSetDomain: (xDomain: [number, number]) => void
```

Comment:
```suggestion
  setDomain: (xDomain: [number, number]) => void
```

this function sets the domain, right? it's not something that happens in reaction to the domain being set. It's doing the setting, yes?